docs/support_table.md

```diff
@@ -785,7 +785,7 @@ use `\ce` instead|
 |\omicron|$\omicron$||
 |\ominus|$\ominus$||
 |\operatorname|$\operatorname{asin} x$|\operatorname{asin} x|
-|\operatorname\*|$\operatorname{asin}\limits_y x$|`\operatorname{asin}\limits_y x`|
+|\operatorname\*|$\operatorname*{asin}\limits_y x$|`\operatorname*{asin}\limits_y x`|
```

Comment:
This example seems to work just as well without `\limits`. Is it worth including `\limits` in the example? (I'm not sure I understand the difference.)

Reply:
> This example seems to work just as well without `\limits`.

To see the difference, you have to be in text mode, not display mode. So the difference will be visible in the documentation page. Let's leave it in.